Makefile

```diff
@@ -264,3 +266,11 @@ check-for-subheading-anchors:
 	$S grep -r -i --include \*.html -L 'Note: this file exempt from check-for-subheading-anchors check' _site/ \
 	  | xargs grep '<h[23456]' \
 	  | grep -v '<h[23456][^>]* id=' | eval $(ERROR_ON_OUTPUT)
+
+check-for-javascript-in-svgs:
+## Security check: don't allow any SVGs that contain Javascript.
+	$S find _site/ -name '*.svg' | xargs grep '<script' | eval $(ERROR_ON_OUTPUT)
+
+check-for-english-in-en-dir:
+## All pages must have page.lang set to work properly with the site templates
+	$S grep -rl -- '---' en/ | xargs grep -L '^[^#]*lang: en' | eval $(ERROR_ON_OUTPUT)
```

Comment:
Oh, duh, that is obviously much better.  I don't know what I was thinking.  I'll update, thanks!